migrations/227_grading_method_split.sql

```diff
@@ -0,0 +1,23 @@
+-- Add new grading columns
+ALTER TABLE submissions 
+    ADD COLUMN IF NOT EXISTS grading_method_internal boolean DEFAULT True,
+    ADD COLUMN IF NOT EXISTS grading_method_external boolean DEFAULT False,
```

Comment:
TO DO: Review if submission level checks are used, as grading_jobs and question level information already exists.